pkg/server/server.go

```diff
@@ -23,8 +23,15 @@ type RegistrationServer struct {
 
 // New creates a new RegistrationServer object with reasonable defaults.
 func New(config *configuration.Config) *RegistrationServer {
+
+	ginRouter := gin.New()
+	ginRouter.Use(
+		gin.LoggerWithWriter(gin.DefaultWriter, "/api/v1/health"),
+		gin.Recovery(),
+	)
```

Comment:
What exactly does this code do? I guess you need some special logging for that endpoint. Can you add some comments? What and why you are doing this.